examples/combobox/css/combobox-autocomplete.css

```diff
@@ -30,14 +31,33 @@
   width: 1.25rem;
   border-left: none;
   outline: none;
-  font-size: 70%;
+}
+
+.combobox .group button.open svg {
```

Comment:
Why?

Reply:
It's a common best practice for efficiency (browser go right-to-left in interpreting CSS selectors), and to avoid an arms race of over-specific CSS selectors.

for reference:
- https://primer.style/css/principles/scss#lint-rules-wip
- https://csswizardry.com/2011/09/writing-efficient-css-selectors/

Reply:
OK, thanks. For efficiency, I think it's better to use child selector over descendant selector, but that would add even more specificity and result in harder to read selectors. I think the selectors here can remove some middle part without changing what matches, so I'll try that.